packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js

```diff
@@ -22,11 +22,15 @@ import React from 'react';
 import { cssModules } from 'bpk-react-utils';
 import BpkText from 'bpk-component-text';
 import BpkButton from 'bpk-component-button';
-import BpkCard from 'bpk-component-card';
 
-import STYLES from './BpkGraphicPromo.module.scss';
+import CARD_STYLES from '../../bpk-component-card/src/BpkCard.module.scss';
 
-const getClassName = cssModules(STYLES);
+import GP_STYLES from './BpkGraphicPromo.module.scss';
+
+const getClassName = cssModules(GP_STYLES);
+const getCardClassName = cssModules(CARD_STYLES);
+
+const ACCESSIBILITY_KEYS = [13, 32];
```

Comment:
nit: should we comment here that these are enter and space button keys?

Reply:
I had it below and then I extracted them and forgot. Good point, adding it right away.

Reply:
Can we turn this into an object instead similar to other components, this then avoids the need for a comment and lays out what they are

```suggestion
const ACCESSIBILITY_KEYS = {
  "key": 13,
  "otherKey": 32
};
```